### DISCUSSION TITLE
Open Telemetry - Metrics support

### DISCUSSION THREAD

**Discussion by kmualem** [kamon-io/Kamon#1310] [Q&A]
Hi,
We currently have a Scala service that uses Kamon with StatsD for metrics.
We would like to move our metrics from StatsD to Open Telemetry.
I saw that there is a library module **kamon-opentelemetry** for trace/spans but I'm not sure this provides us the replacement for the reporter metrics of StatsD.

I tried to find the relevant configuration needed for the open telemetry (like we have for StatsD), but didn't find any resources on the web.

Can you please help?

Thanks

**Comment by hughsimpson** [comment]
Hey! Sorry I missed this, I forgot GitHub even has this QA section... I do have an open pr for otel metrics support but I haven't had time to properly finish it up. Perhaps you could take a look there?
